src/favorite.html

```diff
@@ -5,6 +5,19 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
+    <link rel="stylesheet" href="./sass/index.scss" />
   </head>
-  <body></body>
-</html>
+  <body data-name="favorite">
+   
+
+    <include src="./partials/header.html"></include>
+    <ul class="gallery">
+
+    </ul>
+
+    <include src="./partials/footer.html"></include>
+    <script src="index.js" type="module"></script>
+    <script src="./js/navigation.js"></script>
+     <script src="./js/components/favorite.js" type="module"></script>
+  </body>
+</html> -->
```

Comment:
Забери, будь ласка, комент ``
```suggestion
</html>
```

Reply:
це я вже пофіксила.